src/Form/Type/CustomUrlType.php

```diff
@@ -40,7 +40,27 @@ public function buildForm(FormBuilderInterface $builder, array $options)
             ->add('url', TextType::class, [
                 'label' => $this->trans('URL', 'Modules.Linklist.Admin'),
                 'required' => true,
-                'constraints' => [new Assert\Url()],
+                // Regex adapted from src/Symfony/Component/Validator/Constraints/UrlValidator.php (symfony 3.4)
+                // In symfony 4.4 this would be as simple as `new Assert\Regex(['relativeProtocol' => true])`
+                'constraints' => [new Assert\Regex(['pattern' => '~^
```

Comment:
It's maybe easier to remove the compatibility from PrestaShop 1.7 and only consider PrestaShop 8.0 compatible?

Reply:
Now, after the latest changes, it is well compatible with PS 1.7 and it includes the improvement from symfony 4.4